gui/slick/views/apiBuilder.mako

```diff
@@ -57,9 +42,9 @@
     <link rel="stylesheet" type="text/css" href="css/browser.css?${sbPID}" />
     <link rel="stylesheet" type="text/css" href="css/lib/jquery-ui-1.10.4.custom.min.css?${sbPID}" />
     <link rel="stylesheet" type="text/css" href="css/style.css?${sbPID}"/>
-    <link rel="stylesheet" type="text/css" href="css/${sbThemeName}.css?${sbPID}" />
+    <link rel="stylesheet" type="text/css" href="css/${sickbeard.THEME_NAME}.css?${sbPID}" />
 </head>
-<body>
+<body ${('data-controller="' + controller + '" data-action="' + action + '" api-key="' + sickbeard.API_KEY +'"  api-root="api/v2/"', '')[title == 'Login']}>
```

Comment:
This is needed for core.js to load the info params. The same is on the main.mako file.